content/300-guides/050-database/400-prototyping-schema-db-push.mdx

```diff
@@ -92,20 +93,18 @@ The following scenario demonstrates how to use `db push` to synchronize a new sc
 
 4.  Make an additive change - for example, create a new required field:
 
-    ```prisma highlight=6;add
+    ```prisma highlight=5;add
     /* ... */
-
     model Post {
```

Comment:
This change is a bit unfortunate - probably because Prisma itself does not really understand `/* ... */` even I guess?